### DISCUSSION TITLE
Are there any definitive timelines to add support for Java 21?

### DISCUSSION THREAD

**Discussion by ieshaan12** [robolectric/robolectric#9860] [Q&A]
Basically the title.

**Comment by MGaetan89** [comment]
Could you clarify your request?
Are you having issues using Robolectric with Java 21? Or are you expecting some features?

**Reply by ieshaan12** [reply]
I'm having some issues with Java 21. I didn't create a fresh issue, as there's no official support for 21 from Robolectric, but I just wanted to ask if there's a date for it.

**Reply by MGaetan89** [reply]
Feel free to open an issue for your problem, so we can have a look at it.
I have no idea if there is a plan to support Java 21 at the moment.
